src/SFA.DAS.ApprenticeFeedback.Domain/Entities/FeedbackTransaction.cs

```diff
@@ -5,12 +5,27 @@ namespace SFA.DAS.ApprenticeFeedback.Domain.Entities
     public class FeedbackTransaction
     {
         public Guid Id { get; set; }
-        public Guid ApprenticeFeedbackTargetId { get; set; }
+        public Guid? ApprenticeFeedbackTargetId { get; set; }
```

Comment:
I'd be tempted to reverse this nullable Guid. You can't have a feedback transaction without an apprentice feedback target, even in it's skeleton state.

Reply:
I think this was a temporary change I made to try and get EF code working and forgot to change back. Changed it back now.